google-cloud-spanner/src/main/java/com/google/cloud/spanner/BuiltInMetricsProvider.java

```diff
@@ -80,15 +90,46 @@ OpenTelemetry getOrCreateOpenTelemetry(
     }
   }
 
-  Map<String, String> createClientAttributes(String projectId, String client_name) {
+  void enableGrpcMetrics(
+      InstantiatingGrpcChannelProvider.Builder channelProviderBuilder,
+      String projectId,
+      @Nullable Credentials credentials,
+      @Nullable String monitoringHost) {
+    GrpcOpenTelemetry grpcOpenTelemetry =
+        GrpcOpenTelemetry.newBuilder()
+            .sdk(this.getOrCreateOpenTelemetry(projectId, credentials, monitoringHost))
+            .enableMetrics(BuiltInMetricsConstant.GRPC_METRICS_TO_ENABLE)
+            .disableMetrics(BuiltInMetricsConstant.GRPC_METRICS_ENABLED_BY_DEFAULT)
```

Comment:
Add a comment to clarify that these are metrics enabled by grpc open telemetry itself and we currently don't need them.

Reply:
done